mscr-ui/src/modules/crosswalk-editor/mappings-accordion/index.tsx

```diff
@@ -88,6 +94,14 @@ function Row(props: {
     );
   };
 
+  function performDeleteMappingAction() {
+      setIsDeleteMappingConfirmModalOpen(false);
```

Comment:
The proper indentation is 2 spaces, this block has 4 spaces. I know this file is full of formatting issues, but we can at least try keep our new commits clean. I'm holding on to hope some day we can fix all these.

Reply:
I do not know correct indentation, because it is not mentioned or monitored, but on next commit indentation will be two spaces.

Reply:
Yes, would be nice to add some monitoring. I have Prettier with this in the .prettierrc.json:
{
  "singleQuote": true,
  "tabWidth": 2
}
which helps a little with getting it right automatically.